src/main/java/com/auth0/client/mgmt/filter/QueryFilter.java

```diff
@@ -10,7 +10,7 @@
     /**
      * Filter by a query
      *
-     * @param query the query expression using the following syntax https://auth0.com/docs/api/management/v2/query-string-syntax.
+     * @param query the query expression to use
```

Comment:
this filter applies both for Users and Logs. No need to mention the User syntax here.